.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
empty config file to let editors and other tools know you are using Prettier

https://prettier.io/docs/en/install.html

Reply:
what about [`.prettierrc`](https://github.com/lingui/js-lingui/blob/main/.prettierrc)?

Reply:
Fixed